pkg/operator/controller/ingress/deployment.go

```diff
@@ -1678,3 +1669,27 @@ func GetMIMETypes(mimeTypes []operatorv1.CompressionMIMEType) []string {
 
 	return mimes
 }
+
+// caps the value of ReloadInterval between the bounds of 1s and 120s
+// returns the default of 5s if the user gives a 0 value
+func capReloadIntervalValue(interval time.Duration) time.Duration {
+	const (
+		maxInterval     = 120 * time.Second
+		minInterval     = 1 * time.Second
+		defaultInterval = 5 * time.Second
+	)
```

Comment:
```suggestion
	maxInterval     := 120 * time.Second
	minInterval     := 1 * time.Second
	defaultInterval := 5 * time.Second
	
```

Reply:
What's the reason to make them non-const? I would have thought immutability would be preferable.

Reply:
I thought it was weird, albeit not incorrect, to declare a `const` within the same function it is used.  On second thought I think it's probably better just to return the duration value if the const isn't used outside this function.